Expensify/react-native-live-markdown
# Change expensify-common patch to remove `<emoji>` & `</emoji>` from `data-code-raw`

### Details

This PR fixes the problem where emoji in code block breaks the functionality

### Related Issues

GH_LINK

### Manual Tests

1. Write ` ```test😁``` ` in the input - make sure no warnings in the console appear
2. Now try adding it in separate lines with linebreaks between ` ``` `, `test😁` & ` ``` `

<details>
<summary>Videos</summary>


https://github.com/Expensify/react-native-live-markdown/assets/41169160/3e7bf927-ad3e-4efa-80f2-667d94b94af7

https://github.com/Expensify/react-native-live-markdown/assets/41169160/b71456a7-0189-477c-8baf-655ffa93204d



</details>

### Linked PRs
